example/example.json

```diff
@@ -102,7 +102,7 @@
             "type": "string",
             "title": "Comments",
             "ui:widget": "textarea",
-            "ui:height": 5     
+            "ui:maxlength": 10
```

Comment:
See JSON specs here: http://json-schema.org/latest/json-schema-validation.html#rfc.section.6.3.1
this validation property should be camelCased.